core/src/main/java/org/apache/calcite/rel/rules/SubQueryRemoveRule.java

```diff
@@ -552,6 +552,7 @@ private static RexNode rewriteUnique(RexSubQuery e, RelBuilder builder) {
     return builder.call(SqlStdOperatorTable.NOT, RexSubQuery.exists(relNode));
   }
 
+
```

Comment:
Nice job! I also learned this pr and thought it is OK, but this blank line should be deleted?